<!doctype html><html lang=en><head><meta charset=utf-8><meta name=description content="Heterofermentative (only lactic acid) and homofermentative (lactic acid + ethanol + CO2) anaerob, but aero-tolerant Lactobacillus, Leuconostoc, Pedicococcus, Lactococcus, Streptococcus, Carnobacterium, Enterococcus, Oenococcus, Tetragenococcus, Vagococcus, Weisella They have limited biosynthetic ability, having evolved in environments that are rich in amino acids, vitamins, purines and pyrimidines, so they must be cultivated in complex media that fulfill all their nutritional requirements."><title>Lactic acid bacteria</title><meta name=viewport content="width=device-width,initial-scale=1"><link rel="shortcut icon" type=image/png href=https://silfaz.github.io/digitalgarden//icon.png><link href=https://silfaz.github.io/digitalgarden/styles.591589daec716a7d5287f8d56c2c091e.min.css rel=stylesheet><link href=https://silfaz.github.io/digitalgarden/styles/_light_syntax.86a48a52faebeaaf42158b72922b1c90.min.css rel=stylesheet id=theme-link><script src=https://silfaz.github.io/digitalgarden/js/darkmode.5f4efd09c6dd84a75c2c8b867bc54645.min.js></script>
<link rel=stylesheet href=https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/katex.min.css integrity=sha384-R4558gYOUz8mP9YWpZJjofhk+zx0AS11p36HnD2ZKj/6JR5z27gSSULCNHIRReVs crossorigin=anonymous><script defer src=https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/katex.min.js integrity=sha384-z1fJDqw8ZApjGO3/unPWUPsIymfsJmyrDVWC8Tv/a1HeOtGmkwNd/7xUS0Xcnvsx crossorigin=anonymous></script>
<script defer src=https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/contrib/auto-render.min.js integrity=sha384-+XBljXPPiv+OzfbB3cVmLHf4hdUFHlWNZN5spNQ7rmHTXpd7WvJum6fIACpNNfIR crossorigin=anonymous></script>
<script src=https://unpkg.com/@floating-ui/core@0.7.3></script>
<script src=https://unpkg.com/@floating-ui/dom@0.5.4></script>
<script src=https://silfaz.github.io/digitalgarden/js/popover.37b1455b8f0603154072b9467132c659.min.js></script>
<script src=https://silfaz.github.io/digitalgarden/js/code-title.b35124ad8db0ba37162b886afb711cbc.min.js></script>
<script src=https://silfaz.github.io/digitalgarden/js/clipboard.c20857734e53a3fb733b7443879efa61.min.js></script>
<script>const BASE_URL="https://silfaz.github.io/digitalgarden/",fetchData=Promise.all([fetch("https://silfaz.github.io/digitalgarden/indices/linkIndex.219d513b96fa14a5020905866e525928.min.json").then(e=>e.json()).then(e=>({index:e.index,links:e.links})),fetch("https://silfaz.github.io/digitalgarden/indices/contentIndex.f18db46f495308bb3fa98e5076e5a63d.min.json").then(e=>e.json())]).then(([{index:e,links:t},n])=>({index:e,links:t,content:n})),render=()=>{const n=new URL(BASE_URL),s=n.pathname,o=window.location.pathname,i=s==o;addCopyButtons(),addTitleToCodeBlocks();const e=document.getElementById("graph-container");if(!e)return requestAnimationFrame(render);e.textContent="";const t=i&&!1;drawGraph("https://silfaz.github.io/digitalgarden",t,[{"/moc":"#4388cc"}],t?{centerForce:1,depth:-1,enableDrag:!0,enableLegend:!1,enableZoom:!0,fontSize:.5,linkDistance:1,opacityScale:3,repelForce:1,scale:1.4}:{centerForce:1,depth:1,enableDrag:!0,enableLegend:!1,enableZoom:!0,fontSize:.6,linkDistance:1,opacityScale:3,repelForce:2,scale:1.2}),initPopover("https://silfaz.github.io/digitalgarden",!0,!0)},init=(e=document)=>{addCopyButtons(),addTitleToCodeBlocks(),renderMathInElement(e.body,{delimiters:[{left:"$$",right:"$$",display:!0},{left:"$",right:"$",display:!1}],throwOnError:!1})}</script><script type=module>
    import { attachSPARouting } from "https:\/\/silfaz.github.io\/digitalgarden\/js\/router.9d4974281069e9ebb189f642ae1e3ca2.min.js"
    attachSPARouting(init, render)
  </script></head><script async src="https://www.googletagmanager.com/gtag/js?id=G-XYFD95KB4J"></script>
<script>var doNotTrack=!1;if(!doNotTrack){window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments)}gtag("js",new Date),gtag("config","G-XYFD95KB4J",{anonymize_ip:!1})}</script><body><div id=search-container><div id=search-space><input autocomplete=off id=search-bar name=search type=text aria-label=Search placeholder="Search for something..."><div id=results-container></div></div></div><script src=https://cdn.jsdelivr.net/npm/flexsearch@0.7.21/dist/flexsearch.bundle.js integrity="sha256-i3A0NZGkhsKjVMzFxv3ksk0DZh3aXqu0l49Bbh0MdjE=" crossorigin=anonymous defer></script>
<script defer src=https://silfaz.github.io/digitalgarden/js/search.cf33b507388f3dfd5513a2afcda7af41.min.js></script><div class=singlePage><header><h1 id=page-title><a href=https://silfaz.github.io/digitalgarden/>ðŸŒ³ Silfa's digital garden</a></h1><div class=spacer></div><div id=search-icon><p>Search</p><svg tabindex="0" aria-labelledby="title desc" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 19.9 19.7"><title id="title">Search Icon</title><desc id="desc">Icon to open search</desc><g class="search-path" fill="none"><path stroke-linecap="square" d="M18.5 18.3l-5.4-5.4"/><circle cx="8" cy="8" r="7"/></g></svg></div><div class=darkmode><input class=toggle id=darkmode-toggle type=checkbox tabindex=-1>
<label id=toggle-label-light for=darkmode-toggle tabindex=-1><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="dayIcon" viewBox="0 0 35 35" style="enable-background:new 0 0 35 35"><title>Light Mode</title><path d="M6 17.5C6 16.672 5.328 16 4.5 16h-3C.672 16 0 16.672.0 17.5S.672 19 1.5 19h3C5.328 19 6 18.328 6 17.5zM7.5 26c-.414.0-.789.168-1.061.439l-2 2C4.168 28.711 4 29.086 4 29.5 4 30.328 4.671 31 5.5 31c.414.0.789-.168 1.06-.44l2-2C8.832 28.289 9 27.914 9 27.5 9 26.672 8.329 26 7.5 26zm10-20C18.329 6 19 5.328 19 4.5v-3C19 .672 18.329.0 17.5.0S16 .672 16 1.5v3C16 5.328 16.671 6 17.5 6zm10 3c.414.0.789-.168 1.06-.439l2-2C30.832 6.289 31 5.914 31 5.5 31 4.672 30.329 4 29.5 4c-.414.0-.789.168-1.061.44l-2 2C26.168 6.711 26 7.086 26 7.5 26 8.328 26.671 9 27.5 9zM6.439 8.561C6.711 8.832 7.086 9 7.5 9 8.328 9 9 8.328 9 7.5c0-.414-.168-.789-.439-1.061l-2-2C6.289 4.168 5.914 4 5.5 4 4.672 4 4 4.672 4 5.5c0 .414.168.789.439 1.06l2 2.001zM33.5 16h-3c-.828.0-1.5.672-1.5 1.5s.672 1.5 1.5 1.5h3c.828.0 1.5-.672 1.5-1.5S34.328 16 33.5 16zM28.561 26.439C28.289 26.168 27.914 26 27.5 26c-.828.0-1.5.672-1.5 1.5.0.414.168.789.439 1.06l2 2C28.711 30.832 29.086 31 29.5 31c.828.0 1.5-.672 1.5-1.5.0-.414-.168-.789-.439-1.061l-2-2zM17.5 29c-.829.0-1.5.672-1.5 1.5v3c0 .828.671 1.5 1.5 1.5s1.5-.672 1.5-1.5v-3C19 29.672 18.329 29 17.5 29zm0-22C11.71 7 7 11.71 7 17.5S11.71 28 17.5 28 28 23.29 28 17.5 23.29 7 17.5 7zm0 18c-4.136.0-7.5-3.364-7.5-7.5s3.364-7.5 7.5-7.5 7.5 3.364 7.5 7.5S21.636 25 17.5 25z"/></svg></label><label id=toggle-label-dark for=darkmode-toggle tabindex=-1><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="nightIcon" viewBox="0 0 100 100" style="enable-background='new 0 0 100 100'"><title>Dark Mode</title><path d="M96.76 66.458c-.853-.852-2.15-1.064-3.23-.534-6.063 2.991-12.858 4.571-19.655 4.571C62.022 70.495 50.88 65.88 42.5 57.5 29.043 44.043 25.658 23.536 34.076 6.47c.532-1.08.318-2.379-.534-3.23-.851-.852-2.15-1.064-3.23-.534-4.918 2.427-9.375 5.619-13.246 9.491-9.447 9.447-14.65 22.008-14.65 35.369.0 13.36 5.203 25.921 14.65 35.368s22.008 14.65 35.368 14.65c13.361.0 25.921-5.203 35.369-14.65 3.872-3.871 7.064-8.328 9.491-13.246C97.826 68.608 97.611 67.309 96.76 66.458z"/></svg></label></div></header><article><h1>Lactic acid bacteria</h1><p class=meta>Last updated
Mar 19, 2023</p><ul class=tags></ul><aside class=mainTOC><details><summary>Table of Contents</summary><nav id=TableOfContents><ol><li><a href=#fermentation-pathways>Fermentation pathways</a><ol><li><a href=#homofermentative>Homofermentative</a></li><li><a href=#heterofermentative>Heterofermentative</a></li></ol></li><li><a href=#proteolytic-system>Proteolytic system</a></li><li><a href=#lactobacillaceae>Lactobacillaceae</a><ol><li><a href=#group-1>Group 1</a></li><li><a href=#group-2>Group 2</a></li><li><a href=#group-3>Group 3</a></li><li><a href=#lactobacillus>Lactobacillus</a></li><li><a href=#lacticaseibillus>Lacticaseibillus</a></li></ol></li><li><a href=#different-species-of-_lactobacillaceae_>Different species of <em>Lactobacillaceae</em></a><ol><li><a href=#_lactobacillus-delbrueckii-subsp-bulgaricus_><em>Lactobacillus delbrueckii subsp. bulgaricus</em></a></li><li><a href=#_lactobacillus-delbrueckii-subsp-delbrueckii_><em>Lactobacillus delbrueckii subsp. delbrueckii</em></a></li><li><a href=#_lactobacillus-delbrueckii-subsp-lactis_><em>Lactobacillus delbrueckii subsp. lactis</em></a></li><li><a href=#_lactobacillus-acidophilus_><em>Lactobacillus acidophilus</em></a></li><li><a href=#lactobacillus-helveticus>Lactobacillus helveticus</a></li><li><a href=#_streptococcus-thermophilus_><em>Streptococcus thermophilus</em></a></li><li><a href=#_lactococcus-lactis_><em>Lactococcus lactis</em></a></li><li><a href=#lactococcus-genome>Lactococcus genome</a></li></ol></li><li><a href=#bifidobacterium>Bifidobacterium</a></li><li><a href=#probiotics>Probiotics</a></li><li><a href=#dairy-fermentation>Dairy fermentation</a></li><li><a href=#vegetable-fermentation>Vegetable fermentation</a></li><li><a href=#bacteriocins-and-antibiotics-from-lab>Bacteriocins and antibiotics from LAB</a></li><li><a href=#sources>Sources</a></li></ol></nav></details></aside><ul><li>Heterofermentative (only lactic acid) and homofermentative (lactic acid + ethanol + CO2)</li><li>anaerob, but aero-tolerant</li><li><em>Lactobacillus, Leuconostoc, Pedicococcus, Lactococcus, Streptococcus, Carnobacterium, Enterococcus, Oenococcus, Tetragenococcus, Vagococcus, Weisella</em></li><li>They have limited biosynthetic ability, having evolved in environments that are rich in amino acids, vitamins, purines and pyrimidines, so they must be cultivated in complex media that fulfill all their nutritional requirements. Limited capacity to synthesize amino acids using inorganic nitrogen sources. They are therefore dependent on preformed amino acids being present in the growth medium as a source of nitrogen.</li><li>Contribute to the taste and texture of fermented products and inhibit food spoilage bacteria by <strong>producing growth-inhibiting substances</strong> and <strong>large amounts of lactic acid</strong>.</li><li>Ability to degrade different carbohydrates and related compounds (but depending on species and strain!).</li><li>Spoilage organisms among LAB typically grow slowly with lactic acid as a carbon source.</li></ul><a href=#fermentation-pathways><h2 id=fermentation-pathways><span class=hanchor arialabel=Anchor># </span>Fermentation pathways</h2></a><p>Presence or absence of the key cleavage enzymes of the glycolysis pathway (fructose 1,6-diphosphate) and the pentose phosphate pathway.</p><a href=#homofermentative><h3 id=homofermentative><span class=hanchor arialabel=Anchor># </span>Homofermentative</h3></a><p><strong>Glycolysis pathway</strong>. Some <em>lactobacilli</em> and most species of <em>enterococci</em>, _lactococci, <em>pediococci</em>, <em>streptococci</em>, <em>tetragenococci</em>, and <em>vagococci</em>.</p><ul><li>Under excess gluose and limited oxygen: one mole of <strong>glucose</strong> yields two moles of <strong>pyruvate</strong> (through glycolysis pathway). Oxidation of NADH to NAD, while puryvate gets reduced to <strong>lactate</strong>. The process yields two molecules ATP per glucose consumed.</li><li>Free glucose is transported into the cell and phosphorylated by glucose-6-phosphate by an ATP-dependent hexose kinase. Other sugars (mannose, fructose) enter the major pathway at the level of glucose-6-phosphate or fructore-6-phosphate after isomerization or phosphorylation, or both.</li><li>Alternatively, some species use the phosphoenolpyrucate (PEP) sugar phosphtransferase system (PTS); either for galactose only or for all sugars.</li></ul><a href=#heterofermentative><h3 id=heterofermentative><span class=hanchor arialabel=Anchor># </span>Heterofermentative</h3></a><p><strong>Pentose phosphate pathway</strong>. <em>Leuconostocs</em>, some <em>lactobacilli</em>, <em>oenococci</em>, and <em>weissella</em> species.</p><ul><li>One mole <strong>glucose</strong> &ndash;> <strong>glucose-6-phosphate</strong> &ndash;> dehydrogenated to <strong>6-phosphogluconate</strong> &ndash;> decarboxylated to yield one <strong>CO2</strong>.</li><li><strong>Pentose-5-phosphate</strong> is cleaved into one molecule of glyceraldehyde phosphate (<strong>GAP</strong>) and one molecule of <strong>acetyl phosphate</strong> &ndash;> GAP is further metabolized into <strong>lactate</strong>; acetyl phosphate is reduced to <strong>ethanol</strong> via acetyl-CoA and acetaldehyade intermediates.</li></ul><a href=#proteolytic-system><h2 id=proteolytic-system><span class=hanchor arialabel=Anchor># </span>Proteolytic system</h2></a><p>Since the quantities of free amino acids present in their environment are not sufficient to support the growth of bacteria to a high cell density, they require a proteolytic system capable of hydrolyzing peptides and proteins in order to obtain essential amino acids. All dairy lactococci used for acidification of milk (e.g., in cheese manufacture) have proteolytic activity. The lactococcal proteolytic system consists of enzymes outside the cytoplasmic membrane, transport systems, and intracellular peptidases. The proteolytic activity of LAB contributes additionally to the development of the flavor, aroma and texture of fermented products. For many varieties of cheeses, such as Swiss and Cheddar, desirable &ldquo;flavor tones&rdquo; are derived by proteolysis.</p><a href=#lactobacillaceae><h2 id=lactobacillaceae><span class=hanchor arialabel=Anchor># </span>Lactobacillaceae</h2></a><ul><li>3 major subgroups, >250 species of which 84 are on the International Dairy Federation list of food cultures and around 20 are produced as starter culture or probiotic cultures.</li><li>New taxonomy since 2020: reclassification into 26 genera: <em>Lactobacillus</em>, <em>Pedicococcus</em>, <em>Paralactobacillus</em> and 23 novel genera. 16S rRNA similarity allows robust assignment to the new genera.</li></ul><p><img src=https://silfaz.github.io/digitalgarden//projects/attachments/Pasted%20image%2020230226204406.png width=auto alt>
<em>(Reclassified genera of lactobacillaceae and their translations. Source: Michael GÃ¤nzle talk for The Fermentation Association
<a href="https://www.youtube.com/watch?v=ErQbie690po%29" rel=noopener>https://www.youtube.com/watch?v=ErQbie690po)</a></em></p><ul><li><p>Convertion of old to new names:
<a href=http://www.lactobacillus.uantwerpen.be/ rel=noopener>http://www.lactobacillus.uantwerpen.be/</a></p></li><li><p>More resistant to acidic conditions than other LAB. Can grow below pH 4 &ndash;> often responsible for the final stages of lactic acid fermentation.</p></li><li><p>Very different characteristics among lactobacilli:</p><ul><li>Cocci or rods, may form chains, pairs, tetrads</li><li>Respiring or non-respiring (facultative or stricly anaerobic)</li><li>Homo- or heterofermentative</li><li>Grow at 4ÂºC or at 45ÂºC, or at both temperatures</li><li>Fermenting hexoses, pentoses or both</li><li>Producing or consuming lactate</li></ul></li><li><p><strong>Safety</strong>: A few species can cause rare infections in critically ill patients, mostly in Weissella genus.</p></li><li><p><strong>Antibiotic resistance</strong>: <em>Lactobacillus</em>, <em>Amylolactobacillus</em> and <em>Holzapfelia</em> are vancomycin sensitive, all other genera of <em>Lactobacillaceae</em> are vancomycin resistant. Some strains of intestinal <em>Lactobacillaceae</em> picked up tetracycline resistance.</p></li><li><p>Main product of fermentative metabolism is lactate, other products may be acetate, ethanol, CO2, formate or succinate.</p></li></ul><a href=#group-1><h3 id=group-1><span class=hanchor arialabel=Anchor># </span>Group 1</h3></a><p>Homofermentative, produce lactic acid as major end product (>85%) from glucose. E.g. <em>L. delbrueckii</em> and <em>L. acidophilus</em>. Grow at 45ÂºC but not 15ÂºC.</p><a href=#group-2><h3 id=group-2><span class=hanchor arialabel=Anchor># </span>Group 2</h3></a><p>Homofermentative. Grow at 15ÂºC and show variable growth at 45ÂºC. E.g. <em>L. casei</em> and <em>L. plantarum</em>. Can produce more oxidized fermentations (e.g. acetate) when O2 is present.</p><a href=#group-3><h3 id=group-3><span class=hanchor arialabel=Anchor># </span>Group 3</h3></a><p>Heterofermentative. From glucose they produce lactic acid, ethanol and CO2. E.g. <em>L. fermentum</em> (now called <em>Limosilactobacillus fermentum</em>), <em>L. brevis</em> (<em>Levilactobacillus brevi</em>) and <em>L. kefiri</em> (<em>Lentilactobacillus kefiri)</em>.</p><a href=#lactobacillus><h3 id=lactobacillus><span class=hanchor arialabel=Anchor># </span>Lactobacillus</h3></a><ul><li>Homofermentative</li><li>Most species don&rsquo;t ferment pentoses, but they have the ability to ferment a relatively broad spectrum of carbohydrates (e.g. mannitol), and even fructans, starch or glycogen. Metabolic focus on lactose explains the dominance in yogurt and cheese fermentations (esp. of <em>L. delbrueckii</em>), as well as the presence in the intestines of suckling pigs.</li></ul><a href=#lacticaseibillus><h3 id=lacticaseibillus><span class=hanchor arialabel=Anchor># </span>Lacticaseibillus</h3></a><ul><li>Homofermentative</li><li>Some species ferment pentoses via the phosphoketolase pathway.</li><li>Several species of it are used as dairy starter cultures and probiotics.</li></ul><a href=#different-species-of-_lactobacillaceae_><h2 id=different-species-of-_lactobacillaceae_><span class=hanchor arialabel=Anchor># </span>Different species of <em>Lactobacillaceae</em></h2></a><a href=#_lactobacillus-delbrueckii-subsp-bulgaricus_><h3 id=_lactobacillus-delbrueckii-subsp-bulgaricus_><span class=hanchor arialabel=Anchor># </span><em>Lactobacillus delbrueckii subsp. bulgaricus</em></h3></a><p>Previously <em>L. bulgaricus</em>. Ferments glucose to D(-)-lactic acid and also metabolises fructose, mannose and lactose, but not sucrose. Important starter culture for yogurt and cheese. Type strain: ATCC 11842 (DSM 20081). Genome sequence accession number: JQAV00000000.</p><a href=#_lactobacillus-delbrueckii-subsp-delbrueckii_><h3 id=_lactobacillus-delbrueckii-subsp-delbrueckii_><span class=hanchor arialabel=Anchor># </span><em>Lactobacillus delbrueckii subsp. delbrueckii</em></h3></a><p>Ferments glucose to D(-)-lactic acid and also metabolises sucrose, fructose and mannose, but not lactose. Isolated from vegetables and fermented grains. Type strain: ATCC 9649 (DSM 20074). Genome sequence accession number: AY773949.</p><a href=#_lactobacillus-delbrueckii-subsp-lactis_><h3 id=_lactobacillus-delbrueckii-subsp-lactis_><span class=hanchor arialabel=Anchor># </span><em>Lactobacillus delbrueckii subsp. lactis</em></h3></a><p>Includes strains previously designated as <em>L. lactis</em> and <em>L. leichmannii</em>. Ferments glucose do D(-)-lactic acid and also metabolises sucrose, fructose, mannose, maltose and trehalose. Isolated from milk, cheese and grain mash. Type strain: ATCC 12315 (DSM 20072). Genome sequence accession number: AZDE00000000.</p><a href=#_lactobacillus-acidophilus_><h3 id=_lactobacillus-acidophilus_><span class=hanchor arialabel=Anchor># </span><em>Lactobacillus acidophilus</em></h3></a><p>Produces DL-lactic acid from cellobiose, galactose, lactose, maltose, mannose, sucrose and trehalose. Specific strains have been used as probiotics. Isolated from intestinal tract of humans and animals, human mouth, vagina, sourdough and wine. Type strain: ATCC 4356 (DSM 20079). Genome sequence accession number: AZCS00000000.</p><a href=#lactobacillus-helveticus><h3 id=lactobacillus-helveticus><span class=hanchor arialabel=Anchor># </span>Lactobacillus helveticus</h3></a><p>Produces DL-lactic acid from glucose, galactose, lactose, mannose, and trehaolse, but not cellobiose, mannitol, raffinose and sucrose. Isolated from sour milk, cheese (particularly Emmental and GruyÃ¨re cheeses). Type strain: ATCC 15009 (DSM 20075). Genome sequence accession number: AZEK00000000.</p><a href=#_streptococcus-thermophilus_><h3 id=_streptococcus-thermophilus_><span class=hanchor arialabel=Anchor># </span><em>Streptococcus thermophilus</em></h3></a><p><em>Streptococcus thermophilus</em> is an alpha-hemolytic species of the viridans group. The bacterium is found in milk and milk products. It is not a probiotic (it does not survive the stomach) and generally is used in the production of <strong>yogurt</strong> and the manufacture of several types of cheese, especially <strong>Italian and Swiss cheeses</strong>. The organism is a moderate thermophile with an optimal growth rate at <strong>45 Â°C</strong>. Although <em>S. thermophilus</em> is closely related to other pathogenic streptococci (such as <em>S. pneumoniae</em> and <em>S. pyogenes</em>), <em>S. thermophilus</em> is classified as a non-pathogenic species. It is closely related to <em>S. salivarius</em> in the oral cavity.</p><a href=#_lactococcus-lactis_><h3 id=_lactococcus-lactis_><span class=hanchor arialabel=Anchor># </span><em>Lactococcus lactis</em></h3></a><p>Lactococcus is a genus of of LAB with five major species formerly classified as Group N streptococci. The type species for the genus is <em>L. lactis,</em> which has two subspecies, <em>lactis</em> and <em>cremoris</em>. Lactococci differ from other lactic acid bacteria by their pH, salt and temperature tolerances for growth.</p><p><em>Lactococcus lactis</em> is critical for manufacturing cheeses such as <strong>Cheddar</strong>, <strong>cottage cheese</strong>, <strong>cream cheese</strong>, <strong>Camembert</strong>, <strong>Roquefort</strong> and <strong>Brie</strong>, as well as other dairy products like cultured butter, <strong>buttermilk</strong>, <strong>sour cream</strong> and <strong>kefir</strong>. The bacterium can be used in single strain starter cultures, or in mixed strain cultures with other lactic acid bacteria such as Lactobacillus and Streptococcus.</p><p><img src=https://silfaz.github.io/digitalgarden//projects/attachments/Pasted%20image%2020230226130546.png width=auto alt>
<em>(Lactococcus lactis. UW Department of Bacteriology strain LcL325UW. Magnification 20000X. Scanning electron micrograph by Joseph A. Heintz, University of Wisconsin-Madison. From:
<a href=https://textbookofbacteriology.net/lactics_5.html%29 rel=noopener>https://textbookofbacteriology.net/lactics_5.html)</a></em></p><p><img src=https://silfaz.github.io/digitalgarden//projects/attachments/Pasted%20image%2020230226131106.png width=auto alt>
<em>(Lactococcus lactis. Magnification 1500X. Phase micrograph courtesy of T.D. Brock, University of Wisconsin-Madison. From:
<a href=https://textbookofbacteriology.net/lactics_5.html%29 rel=noopener>https://textbookofbacteriology.net/lactics_5.html)</a></em></p><a href=#lactococcus-genome><h3 id=lactococcus-genome><span class=hanchor arialabel=Anchor># </span>Lactococcus genome</h3></a><p>Partly due to their industrial relevance, both <em>Lactococcus lactis</em> subspecies (<em>lactis</em> and <em>cremoris</em>) are widely used as generic LAB models for research. <em>L. lactis ssp. cremoris</em>, used in the production of hard cheeses, is represented by the laboratory strains <strong>LM0230</strong> and <strong>MG1363</strong>.
Currently, there are two <em>L. lactis ssp. cremoris</em> that have been sequenced for public release, one of which is <em>L. lactic ssp. cremoris</em> MG1363
<a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1855848/ rel=noopener>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1855848/</a>.</p><p>Similarly, <em>L. lactis ssp. lactis</em> is employed in soft cheese fermentations, with the workhorse strain <strong>IL1403</strong>, ubiquitous in LAB research laboratories. In 2001, the genome of strain IL1403 was sequenced leading to increased understanding of LAB genomics and related applications.</p><p>The genome sequence reveals 12 enzymes called aminotransferases, some of which are used to break down complex, branched, ring-shaped, and sulfur-containing amino acids. The molecules produced when the amino acids are degraded are very important for cheese flavor. Understanding which amino acids are broken down by which enzymes could give cheese makers greater control over flavor and fragrance of their cheeses.</p><a href=#bifidobacterium><h2 id=bifidobacterium><span class=hanchor arialabel=Anchor># </span>Bifidobacterium</h2></a><p>Bifidobacterium is not included in the traditional Lactic Acid Bacteria due to its genetic unrelatedness, but the bacterium has a habitat that overlaps with LAB, and it has a metabolism that produces lactic acid as a primary end-product of fermentation.Â Bifidobacteria are strictly anaerobic and are a prominant Gram-positve bacterium in the large intestine (colon). <em>Bifidobacterium infantum</em> is the predominant bacterium in the intestine of breast-fed infants because mother&rsquo;s milk contains a specific growth factor that enriches for the growth of the bacterium.</p><p>Among 30 species, those recognized as probiotics include:</p><ul><li><em>B. adolescentis</em></li><li><em>B. breve</em></li><li><em>B. longum</em></li><li><em>B. animalis</em></li><li><em>B. infantis</em></li><li><em>B. thermophilum</em></li><li><em>B. bifidum</em></li><li><em>B. lactis</em></li></ul><a href=#probiotics><h2 id=probiotics><span class=hanchor arialabel=Anchor># </span>Probiotics</h2></a><p><a href=/digitalgarden/projects/fermentation/Pre-pro-and-postbiotics/ rel=noopener class=internal-link data-src=/digitalgarden/projects/fermentation/Pre-pro-and-postbiotics/>Pre-, pro- and postbiotics</a></p><a href=#dairy-fermentation><h2 id=dairy-fermentation><span class=hanchor arialabel=Anchor># </span>Dairy fermentation</h2></a><p>See my & Max Graze&rsquo;s
<a href="https://airtable.com/invite/l?inviteId=inv4NzMfaMoGOAlro&inviteToken=be8dca27eeaaa4e313349e17c353744edd29320c373d962cbea4abf36fcd06dc&utm_medium=email&utm_source=product_team&utm_content=transactional-alerts" rel=noopener>Dairy Ferments Airtable</a></p><p><a href=/digitalgarden/projects/fermentation/Dairy-fermentation-notes/ rel=noopener class=internal-link data-src=/digitalgarden/projects/fermentation/Dairy-fermentation-notes/>Dairy fermentation notes</a></p><p><a href=/digitalgarden/projects/attachments/Plant-based-dairy-fermentation-notes/ rel=noopener class=internal-link data-src=/digitalgarden/projects/attachments/Plant-based-dairy-fermentation-notes/>Plant-based dairy fermentation notes</a></p><a href=#vegetable-fermentation><h2 id=vegetable-fermentation><span class=hanchor arialabel=Anchor># </span>Vegetable fermentation</h2></a><p><a href=/digitalgarden/projects/fermentation/Vegetable-fermentation-notes/ rel=noopener class=internal-link data-src=/digitalgarden/projects/fermentation/Vegetable-fermentation-notes/>Vegetable fermentation notes</a></p><a href=#bacteriocins-and-antibiotics-from-lab><h2 id=bacteriocins-and-antibiotics-from-lab><span class=hanchor arialabel=Anchor># </span>Bacteriocins and antibiotics from LAB</h2></a><p><strong>Nisin</strong> produced by <em>Lactococcus lactis ssp. lactis</em> has been studied extensively. It has a broad spectrum of activity against Gram-positive bacteria. The primary target is believed to be the cell membrane. Unlike some other antimicrobial peptides, nisin does not need a receptor for its interaction with the cell membrane; however, the presence of a membrane potential is required.</p><p>Nisin is a natural preservative present in cheese made with <em>Lactococcus lactis</em> ssp. <em>lactis</em>, but it is also used as a preservative in heat processed and low pH foods. Since nisin cannot be synthesized chemically, the nisin-producing <em>Lactococcus lactis</em> strains are used for its industrial synthesis.</p><p>The first established use of nisin was as a preservative in processed cheese products, but numerous other applications in preservation of foods and beverages have been identified. It is currently recognized as a safe food preservative in approximately 50 countries. Nisin has been used as a preservative in various pasteurized dairy products and canned vegetables, baked, high-moisture flour products, and pasteurized liquid eggs. There is interest in the use of nisin in natural cheese production. Considerable research has been carried out on the anti-listerial properties of nisin in foods and a number of applications have been proposed. Uses of nisin to control spoilage lactic acid bacteria have been identified in beer, wine, alcohol production, and high acid foods such as salad dressings. Production of highly purified nisin preparations has led to interest in the use of nisin for human ulcer therapy and mastitis control in cattle.</p><p>Although nisin is today the only bacteriocin that reached commercial status, approved worldwide as a natural food preservative, many other bacteriocins may soon reach similar status. Several lactobacilli having antibacterial activity against pathogens such as <em>Listeria monocytogenes, Staphyloccus aureus</em>, <em>Salmonella newport</em> and even <em>E.coli</em>.</p><a href=#sources><h2 id=sources><span class=hanchor arialabel=Anchor># </span>Sources</h2></a><ul><li><a href=https://textbookofbacteriology.net/lactics.html rel=noopener>https://textbookofbacteriology.net/lactics.html</a></li><li>Lactic Acid Bacteria: Food Safety and Human Health Applications. Ayivi et al., 2020. dairy</li><li>The Fermentation Association seminar: The new taxonomy of Lactobacillus - Michael GÃ¤nzle & Ben Wolfe. 2021.
<a href="https://www.youtube.com/watch?v=ErQbie690po" rel=noopener>https://www.youtube.com/watch?v=ErQbie690po</a></li><li>2020, Zheng et al. A taxonomic note on the genus Lactobacillus
<a href=https://pubmed.ncbi.nlm.nih.gov/32293557/ rel=noopener>https://pubmed.ncbi.nlm.nih.gov/32293557/</a></li></ul></article><hr><div class=page-end><div class=backlinks-container><h3>Backlinks</h3><ul class=backlinks><li><a href=/digitalgarden/blog/Disovery-learning-and-activity-log/ data-ctx="Lactic acid bacteria" data-src=/blog/Disovery-learning-and-activity-log class=internal-link>Disovery, learning and activity log</a></li><li><a href=/digitalgarden/projects/science/01-science-main/ data-ctx="Lactic acid bacteria" data-src=/projects/science/01-science-main class=internal-link>Science Main</a></li></ul></div></div><div id=contact_buttons><footer><p>Made by Silvia HÃ¼ttner using <a href=https://github.com/jackyzha0/quartz>Quartz</a>, Â© 2024</p><ul><li><a href=https://silfaz.github.io/digitalgarden/>Home</a></li><li><a href=https://twitter.com/silfaz>Twitter</a></li><li><a href=https://github.com/Silfaz>Github</a></li></ul></footer></div></div></body></html>